pom.xml

```diff
@@ -1824,8 +1825,8 @@
         </plugin>
         <plugin>
           <groupId>org.fusesource.hawtjni</groupId>
-          <artifactId>maven-hawtjni-plugin</artifactId>
-          <version>1.14</version>
+          <artifactId>hawtjni-maven-plugin</artifactId>
+          <version>1.18</version>
```

Comment:
Why do we need this change ? I think this is not really related and should be done as a separate PR.

Reply:
You are right, this should be a separate PR.
The issue here is that the plugin requires additional configuration to support newer MSVC versions: https://github.com/fusesource/hawtjni/issues/76.

That configuration property was however only available with the newer hawtjni version, which conveniently was also renamed...

Old versions of visual studio don't seem to be available anymore from microsoft, so I could not compile and test this otherwise.